wandb/sdk/wandb_run.py

```diff
@@ -756,6 +757,10 @@ def restore(
         files = api_run.files([name])
         if len(files) == 0:
             return None
+        # if the file does not exist, the file has an md5 of 0
+        if files[0].md5 == "0":
+            raise ValueError("File {} not found.".format(path))
+            return None
```

Comment:
return is unnecessary

Reply:
Oh, you're right. I thought I was getting an extraComm error. I'll remove that